public/src/js/constants/article-meta-fields.js

```diff
@@ -64,6 +64,30 @@ export default Object.freeze([
         type: 'text'
     },
     {
+        key: 'imageSource',
+        editable: true,
+        dropImage: true,
+        omitForSupporting: true,
+        visibleWhen: 'imageReplace',
+        label: 'replacement image URL',
+        validator: {
+            params: {
+                src: 'imageSrc',
+                width: 'imageSrcWidth',
+                height: 'imageSrcHeight',
+                origin: 'imageSrcOrigin',
+                imageSource: 'imageSource',
+                options: {
+                    maxWidth: 1000,
```

Comment:
Yes, with this change we're simply getting ready to select a different image. Neither of us has figured out yet how to choose the master. We'll keep it for next PR